src/Microsoft.ML.HalLearners/SymSgdClassificationTrainer.cs

```diff
@@ -754,6 +754,11 @@ private void CheckLabel(RoleMappedData examples, out int weightSetCount)
 
         private static unsafe class Native
         {
+            static Native()
+            {
+                var t = ErrorMessage(0);
```

Comment:
You can use:

```
_ = ErrorMessage(0);
```

instead of `var t`.  `_` signifies "I want to capture the variable, but I don't actually use the variable".